ytmusicapi/parsers/search.py

```diff
@@ -125,7 +134,7 @@ def parse_search_result(data, search_result_types, result_type, category):
         search_result["duration"] = None
         search_result["year"] = None
         flex_item = get_flex_column_item(data, 1)
-        runs = flex_item["text"]["runs"][default_offset:]
+        runs = flex_item["text"]["runs"]
```

Comment:
why this change?

Reply:
Like the other, this was part of the issue too

With my change
![image](https://github.com/sigma67/ytmusicapi/assets/10100397/6d1dcc66-e6f1-4e29-ba3c-15e014d68db9)

With original change
![image](https://github.com/sigma67/ytmusicapi/assets/10100397/aafd9e28-3e06-4860-bdfe-fd65194d32a2)

I crossed check the Json files I am writing too and I see with removing this offset it has better chance of showing the artist name with Id over nothing.

Reply:
If you check the before and after jsons I attached to the PR, it will also show this difference